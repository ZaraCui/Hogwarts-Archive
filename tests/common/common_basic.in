ADD COLLECTION tests/data/books_small.csv
ADD STUDENT Harry Potter
ADD STUDENT Hermione Granger
RENT 100000 1001
RENT 100001 1001
RELINQUISH 100000 1001
RELINQUISH 100001 1001
COMMON 100000 100001
EXIT

