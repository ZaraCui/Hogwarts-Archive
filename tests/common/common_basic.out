user: 12 spellbooks successfully added.

user: Success.

user: Success.

user: Success.

user: Spellbook is currently unavailable.

user: Success.

user: Unable to return spellbook.

user: No common spellbooks.

user: Ending Archive process.
